packages/wallets/src/components/TradingAccountCard/TradingAccountCard.scss

```diff
@@ -1,15 +1,25 @@
-$columns: 3;
-$grid-last-row-children: '&:nth-child(#{$columns}n+1):nth-last-child(-n+#{$columns})';
-
 .wallets-trading-account-card {
     display: flex;
     align-items: center;
     gap: 16px;
     height: 88px;
 
-    #{$grid-last-row-children} > &__content,
-    #{$grid-last-row-children} ~ & > &__content {
-        border-bottom: none;
+    @include desktop {
+        $columns: 3;
+        $grid-last-row-children: '&:nth-child(#{$columns}n+1):nth-last-child(-n+#{$columns})';
+        #{$grid-last-row-children} > &__content,
+        #{$grid-last-row-children} ~ & > &__content {
+            border-bottom: none;
+        }
+    }
+
+    @include mobile {
+        $columns: 1;
+        $grid-last-row-children: '&:nth-child(#{$columns}n+1):nth-last-child(-n+#{$columns})';
+        #{$grid-last-row-children} > &__content,
+        #{$grid-last-row-children} ~ & > &__content {
+            border-bottom: none;
```

Comment:
This adds the bottom borders for the last account cards for mobile, while keeping the styling for desktop consistent